From c32e9ffd34c1e77a02ef2fe970faf252ab25a371 Mon Sep 17 00:00:00 2001
From: Zhe Zhang <zhz@apache.org>
Date: Wed, 21 Oct 2015 14:03:12 -0700
Subject: [PATCH 1086/2748] HDFS-9274. Default value of
 dfs.datanode.directoryscan.throttle.limit.ms.per.sec
 should be consistent. Contributed by Yi Liu.

Change-Id: Iafa2efb8f2eafd193b417690cc8794080b9d745b
(cherry picked from commit 2e55226ef5c1634faeccbe1e4b02b78f9a956a86)
---
 .../src/main/resources/hdfs-default.xml            |    2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/hadoop-hdfs-project/hadoop-hdfs/src/main/resources/hdfs-default.xml b/hadoop-hdfs-project/hadoop-hdfs/src/main/resources/hdfs-default.xml
index 35cc707..11da338 100644
--- a/hadoop-hdfs-project/hadoop-hdfs/src/main/resources/hdfs-default.xml
+++ b/hadoop-hdfs-project/hadoop-hdfs/src/main/resources/hdfs-default.xml
@@ -601,7 +601,7 @@
 
 <property>
   <name>dfs.datanode.directoryscan.throttle.limit.ms.per.sec</name>
-  <value>0</value>
+  <value>1000</value>
   <description>The report compilation threads are limited to only running for
   a given number of milliseconds per second, as configured by the
   property. The limit is taken per thread, not in aggregate, e.g. setting
-- 
1.7.9.5

